buildroot-external/kernel/v6.6.y/device-support-pci.config

```diff
@@ -39,3 +39,10 @@ CONFIG_SND_HDA_CODEC_HDMI=m
 # NVMe SSD support
 CONFIG_NVME_CORE=y
 CONFIG_BLK_DEV_NVME=y
+
+# Mellanox ConnectX-3 NIC support
+CONFIG_MLX4_EN=m
+CONFIG_MLX4_EN_DCB=y
```

Comment:
`CONFIG_DCB` is not enabled in our kernel, and most likely won't have any use in HAOS context.
```suggestion
```